### PR TITLE
Utility token contract interact

### LINKED ISSUE #116: Beta.1 issues
- [x] Missing exposed contract interacts:
   - [x] GatewayLib,
   - [x] MerklePatriciaProof,
   - [x] MessageBus,
   - [x] Organization,
   - [x] UtilityToken
- [x] Organization should call `getOrganization` instead of `getEIP20Token `
- [x] Make `admin` optional for organization deployment.

### PR DESCRIPTION
**Pull request by 0xsarvesh**
References #116  
Exposing Utility token contract interact.

### CODE REVIEW

File: src/ContractInteract/UtilityToken.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
+/**
+ * @typedef {Object} UtilityTokenConfig
```
Comment (hobofan) [inline src/ContractInteract/UtilityToken.js]: `UtilityTokenConfig` should be `UtilityTokenSetupConfig`, to be consistent with the other setup configs.

File: src/ContractInteract/UtilityToken.js [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+    this.approveRawTx = this.approveRawTx.bind(this);
+    this.allowance = this.allowance.bind(this);
+    this.isAmountApproved = this.isAmountApproved.bind(this);
+    this.balanceOf = this.balanceOf.bind(this);
+    this.setCoGateway = this.setCoGateway.bind(this);
+    this.setCoGatewayRawTx = this.setCoGatewayRawTx.bind(this);
+  }
+
+  /**
+   * Setup for Utility token contract. Deploys the contract and initializes
```
Comment (hobofan) [inline src/ContractInteract/UtilityToken.js]: Setup for `UtilityToken` doesn't initialize anything. That's only applicable for `OSTPrime`.

**Review by hobofan — APPROVED** [review]
Just some small notes on documentation.

File: test/UtilityToken/allowance.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    let expectedAllowance = '100';
+    const spyContractAllowance = sinon.replace(
+      utilityToken.contract.methods,
+      'allowance',
+      sinon.fake.returns({
+        call: () => Promise.resolve(expectedAllowance),
+      }),
+    );
+
+    const spyOstPrimeAllowance = sinon.spy(utilityToken, 'allowance');
```
Comment (schemar) [inline test/UtilityToken/allowance.js]:
```suggestion
    const spyUtilityTokenAllowance = sinon.spy(utilityToken, 'allowance');
```
?

**Review by schemar — APPROVED** [review]
✈️

**Review by schemar — APPROVED** [review]
:shipit:

**Pull request merged**